src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.ps1

```diff
@@ -12,6 +12,7 @@ foreach ($file in Get-ChildItem $ToolRuntimePath *.runtimeconfig.json)
 {
     Write-Host "Correcting runtime version of" $file.FullName
     $text = (Get-Content $file.FullName) -replace "1.1.0","$highestVersion"
+    $text = (Get-Content $file.FullName) -replace "2.0.0-preview1-001913-00","$highestVersion"
```

Comment:
Instead of this why not make all the versions consistent?

Reply:
I was going to do that but then I saw that you explicitly set this the msbuild.runtimeconfig.json only to this version, which led me to think that you did that for a very specific reason which is why I chose to instead fix the scripts. If there is no particular reason why you updated it on PR: #1460 then I'll just go ahead and move it back to 1.1.0 as well.

Reply:
I actually don't recall that was probably me porting back what was changed in the individual repos. I suspect it doesn't matter as we are updating anyway.